### PR TITLE
fix for sos report failure

### PR DESCRIPTION
**Pull request by rkhambad**
Signed-off-by: Rajendra Khambadkar <rkhambad@rkhambad.remote.csb>

This will fix the sos report failure.

### CODE REVIEW

File: tests/misc_env/sosreport.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def generate_sosreport(cnode, results):
         cnode.exec_command(cmd="sudo yum -y install sos", check_ec=False)
 
         out, err = cnode.exec_command(
-            sudo=True, cmd="sosreport -a --all-logs -e ceph --batch", timeout=1200
+            sudo=True, cmd="sos report -a --all-logs -e ceph --batch", timeout=1200
```
[the remark this answers is not in the document]
Comment (anrao19) [reply tests/misc_env/sosreport.py:50]: And also could you please make changes in https://github.com/red-hat-storage/cephci/blob/master/utility/collect_logs.py#L135

Comment (ckulal) [inline tests/misc_env/sosreport.py:50]: @rkhambad , If it is tested with above changes could you please attach pass logs

[the remark this answers is not in the document]
Comment (rkhambad) [reply tests/misc_env/sosreport.py]: i am really sorry, i need to make this as draft PR to just check, if it affect anything. And i forget to make that and it went open for review. 
Making some new changes and will update the PR.

Comment (rkhambad) [reply tests/misc_env/sosreport.py]: i am really sorry, i need to make this as draft PR to just check, if it affect anything. And i forget to make that and it went open for review.

**Comment by rkhambad** [comment]
success log : https://jenkins.ceph.redhat.com/view/RHCS%20QE/job/rhceph-test-executor/1968/console

**Review by anrao19 — APPROVED** [review]

**Review by sunilkumarn417 — APPROVED** [review]

**Pull request merged**